api/src/customPipe.pipe.ts

```diff
@@ -0,0 +1,68 @@
+import { PipeTransform, Injectable, HttpException, BadRequestException } from '@nestjs/common';
```

Comment:
The naming of the file is weird, `customPipe.pipe.ts` the `pipe.ts` already says that it's a pipe. Also, I prefer it would be better to have one file for each pipe, in a pipes module.